.vscode/launch.json

```diff
@@ -4,32 +4,15 @@
     // Weitere Informationen finden Sie unter https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
-
-        {
-            "name": "[Debug] main",
-            "type": "dart",
-            "request": "launch",
-            "args": [],
-            "program": "app/lib/main.dart"
-        },
-        {
-            "name": "[Release] main",
-            "type": "dart",
-            "request": "launch",
-            "args": [
-                "--release",
-            ],
-            "program": "app/lib/main.dart"
-        },
```

Comment:
We don't need them anymore, because we can now specify the flavors for all platforms via Dart.